monitors/SyntheticPegMonitor.js

```diff
@@ -263,6 +260,14 @@ class SyntheticPegMonitor {
         continue;
       }
 
+      // Set default values for min and max to the most recent non-null price.
+      if (!min) {
```

Comment:
This now will set min and max to the most recent non-null price, allowing for stray null historical prices. Similar to CryptoWatchPricefeed, this method will only return `null` if ALL prices within a lookback window are null.